org.technic93.e2designer.yaml

```diff
@@ -0,0 +1,20 @@
+app-id: "org.technic93.e2designer"
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: e2designer
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
Why does it need host access?

Reply:
I need to open files in the host filesystem for editing, because the application is an editor.
Is it correct usage?